runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -32,6 +32,11 @@ pub mod state {
     pub const CODES: &[u8] = &[0x02];
     /// Prefix for Ethereum account storage in our storage (maps H160||H256 -> H256).
     pub const STORAGES: &[u8] = &[0x03];
+
+    /// Prefix for Ethereum block hashes(only for last 256 blocks excluding current) 
+    /// storage in our storage (maps H160||U256 -> H256).
```

Comment:
Please update the rest of this comment as well (`HASHES` does not map H160||U256 to H256).